### PR TITLE
Add github-actions

### PR DESCRIPTION
**Pull request by HokutoMorita**
Added github-actions for gem push.

Target thread
  - https://prime-number.slack.com/archives/C01R5FL2MFY/p1657514165933819

Parent pull request
  - https://github.com/trocco-io/embulk-output-bigquery/pull/23

### CODE REVIEW

**Comment by HokutoMorita** [comment]
gem pushされることを確認
https://github.com/trocco-io/embulk-output-bigquery_java/actions/runs/2648229849

**Review by kekekenta — APPROVED** [review]

**Comment by HokutoMorita** [comment]
@d-hrs 
こちらご確認よろしくお願いいたします！

**Comment by HokutoMorita** [comment]
@d-hrs 
Please code review

**Review by d-hrs — CHANGES_REQUESTED** [review]
please change title & description in English.

File: build.gradle [hunk 1]
```diff
@@ -12,9 +13,18 @@ repositories {
...
 
 group = "io.trocco"
-version = "0.1.0"
 description = "Bigquery output for embulk"
 
+version = {
+    def baseVersion = "0.1.0"
+    def vd = versionDetails()
+    if (vd.commitDistance == 0 && vd.lastTag ==~ /^v[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
+        baseVersion
```
Comment (d-hrs) [inline build.gradle:22]: do not use `baseVersion`
```suggestion
        vd.lastTag.substring(1)
```

Comment (HokutoMorita) [reply build.gradle:22]: It is a correspondence to link with the release tag.
I got it.

Comment (HokutoMorita) [reply build.gradle]: fixed https://github.com/trocco-io/embulk-output-bigquery_java/pull/94/commits/1c1d6a09dd51f7a2d0dc9b3fd343daa254782f31

File: build.gradle [hunk 2]
```diff
@@ -12,9 +13,18 @@ repositories {
 }
 
 group = "io.trocco"
-version = "0.1.0"
 description = "Bigquery output for embulk"
 
+version = {
+    def baseVersion = "0.1.0"
```
Comment (d-hrs) [inline build.gradle:19]: remove `baseVersion`. use `VersionDetails()` only.

see: https://github.com/palantir/gradle-git-version

File: build.gradle [hunk 3]
```diff
@@ -12,9 +13,18 @@ repositories {
...
+    def vd = versionDetails()
+    if (vd.commitDistance == 0 && vd.lastTag ==~ /^v[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
+        vd.lastTag.substring(1)
+    } else {
+        "${baseVersion}.${vd.gitHash}.pre"
```
Comment (d-hrs) [inline build.gradle:24]:
```suggestion
        "0.0.0.${vd.gitHash}.pre"
```

**Review by d-hrs — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (HokutoMorita) [reply build.gradle]: I got it.

File: build.gradle (hunk 3 above)
Comment (HokutoMorita) [reply build.gradle]: I got it.

**Comment by HokutoMorita** [comment]
@d-hrs
Please code review

**Review by d-hrs — APPROVED** [review]

**Pull request merged**